generators/app/templates/my-awesome-site/gulpfile.babel

```diff
@@ -74,17 +74,19 @@ gulp.task('css', () => {
 <% if (includeSass || includeScss) { -%>
 // Compile scss to css.
 gulp.task('scss', () => {
+//This will check for sass in all pages of the site, incase you use them
+//TODO: Move all Jekyll setup in a folder, build to another folder
```

Comment:
The [Jekyll](https://jekyllrb.com/) way is to build the `_site` (which is the dist directory) to the Jekyll root project. We use the `jekyll build` command, so the behavior is correct. I don't think we need to move the Jekyll to a `src` directory and output the build to a `dist` directory (I think we should keep the Jekyll way as much as we can in this generator), but I'll be happy do discuss it with the community in an issue.

So, please remove this `TODO` for the meantime and open an issue about it so the whole community can share their thoughts about this topic there.